index.js

```diff
@@ -192,27 +194,38 @@ async function checkHeartbeat() {
   const locations = await db.getLocations()
 
   for (const location of locations) {
+    let latestRadar
     // Query raw sensor data to transmit to the FrontEnd
-    const XeThruData = await redis.getLatestXeThruSensorData(location.locationid)
+    if (location.radarType === RADAR_TYPE.XETHRU) {
+      const XeThruData = await redis.getLatestXeThruSensorData(location.locationid)
+      latestRadar = Date(XeThruData.timestamp)
+    } else if (location.radarType === RADAR_TYPE.INNOSENT) {
+      const InnosentData = await redis.getLatestInnosentSensorData(location.locationid)
```

Comment:
This looks strange to me to have these values start with a capital letter. They look like class names.